editor/components/PointMap.tsx

```diff
@@ -140,14 +144,16 @@ export const PointMap = ({
           color={isEditing ? COLORS.danger : undefined}
         />
       )}
-      {/* add nearmap raster source and style */}
-      <MapAerialSourceAndLayer />
```

Comment:
this component was responsible for rendering nearmap aerial imagery as the top layer, in a follow up PR we are going to make it so that the "Aerial" option in all the basemap pickers uses this nearmap imagery but for now i removed this